common/jobLifecycleHandler.go

```diff
@@ -30,6 +30,14 @@ type JobLifecycleHandler interface {
 	OnTransferProgress(progress TransferProgress)
 
 	OnComplete(summary JobSummary)
+
+	// TODO (gapra): Rename these variables at the end. Choosing not to at the moment to limit the scope of changes.
```

Comment:
I meant to type functions - will fix this before merging, but its not a big deal since its just a comment